buildpacks/metrics-agent/src/bin/agentmon_loop.rs

```diff
@@ -0,0 +1,175 @@
+// Enable Clippy lints that are disabled by default.
+// https://rust-lang.github.io/rust-clippy/stable/index.html
+#![warn(clippy::pedantic)]
+
+use clap::Parser;
+use std::ffi::OsStr;
+use std::process::ExitStatus;
+use std::{
+    collections::HashMap,
+    path::{Path, PathBuf},
+    process::{exit, Command},
+    thread::sleep,
+    time::Duration,
+};
+const SLEEP_FOR: Duration = Duration::from_secs(1);
+
+/// Agentmon Loop
+///
+/// Boots agentmon (a statsd server) in a loop
+///
+/// Example:
+///
+/// ```shell
+/// $ cargo run --bin agentmon_loop -- --path <path/to/agentmon/binary>
```

Comment:
Ugh, I wish we had a better way to keep a process running. Really `systemd` is almost the solution (it can start and keep a process running, and is on most linux distros), but it needs to be started under PID 1.